packages/suite/src/components/firmware/CheckSeed.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import { Button, Checkbox, colors } from '@trezor/components';
+import { useDevice, useFirmware } from '@suite-hooks';
+import { Translation } from '@suite-components';
+import { P, H2, WarningImg, SeedImg, ReconnectInNormalStep } from '@firmware-components';
+
+const CheckboxRow = styled.div`
+    display: flex;
+    flex-direction: row;
+    border-top: 1px solid ${colors.NEUE_STROKE_GREY};
+    width: 380px;
+    padding: 20px 0;
+    justify-content: center;
+    margin: 28px auto 0 auto;
+`;
+
+const Body = () => {
+    const { device } = useDevice();
+    const { toggleHasSeed, hasSeed } = useFirmware();
+
+    // ensure that device is connected in requested mode
+    if (device?.mode !== 'normal') return <ReconnectInNormalStep.Body />;
+
+    if (!device?.features?.needs_backup) {
+        return (
+            <>
+                <SeedImg />
+                <H2>
+                    <Translation id="TR_SECURITY_CHECKPOINT_GOT_SEED" />
+                </H2>
+                <P>
+                    <Translation id="TR_BEFORE_ANY_FURTHER_ACTIONS" />
+                </P>
+                <CheckboxRow>
+                    <Checkbox
+                        isChecked={hasSeed}
+                        onClick={toggleHasSeed}
+                        data-test="@firmware/confirm-seed-checkbox"
+                    />
+                    <P>
+                        {' '}
```

Comment:
Personal preference here, I favour the use of `&nbsp;` rather than a string with a space.

Reply:
Heh, I agree.. :D never remember it and I am lazy to google it.